builds/codex-v103-aeturnum-monorepo/tools/semantic_diff.py

```
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)